util/test/loggerTest.js

```diff
@@ -0,0 +1,268 @@
+/* eslint no-underscore-dangle: [2, { "allow": ["__get__", "_logInfo", "_logError",
+"_logDebug"] }] */
+
+/* The writing in log file is async. Hence, even after specifying a wait period,
+there is a chance the tests may fail. */
+
+const chai = require('chai');
+const winston = require('winston');
+const Logger = require('../logger.js').Logger;
+const { exec } = require('child_process');
+const fs = require('fs');
+const chaiFs = require('chai-fs');
+const path = require('path');
+const dirtyChai = require('dirty-chai');
+const rewire = require('rewire');
+const config = require('../../deploy/configs/util/logger.json');
+
+const logDirectory = config.logDirectory;
+const createLoggerObject = rewire('../logger.js').__get__('createLoggerObject');
+
+chai.should();
+chai.use(chaiFs);
+chai.use(dirtyChai);
+
+/* These variables will be used in the following tests.
+They have been declared in the global space to avoid repetition. */
+let logger;
```

Comment:
The names can be MSLogger and LBLogger. Similarly, the logger config objects can be named as MSLoggerConfig and LBLoggerConfig. Such self-descriptive names make the code more readable.